02-regex/README.md

```diff
@@ -76,3 +76,4 @@ _(Please note you are welcome to post under a pseudonym and/or password protect
 
 - Name -- [assignment title](assignment url)
 - David Leyva -- [RegEx Quiz](https://wp.nyu.edu/davidaleyva/week-2-homework/)
+- Soojin Lee -- [Regex Exercise](file:///Users/soojinlee/Desktop/ITPBlog/week2.html)
```

Comment:
Hi Soojin! This link is to your local file I think maybe you meant it to be your uploaded site url?